lib/mutant/mutator/node/send.rb

```diff
@@ -26,8 +26,9 @@ class Send < self
           to_i:          %i[to_int],
           to_a:          %i[to_ary],
           to_h:          %i[to_hash],
-          at:            %i[fetch],
-          :[] =>         %i[at fetch],
+          at:            %i[fetch key?],
+          fetch:         %i[key?],
+          :[] =>         %i[at fetch key?],
```

Comment:
@backus Ideally we do a "bigger" refactoring where a mutation operator only goes "one level", and the mutation engine re-injects the mutant to generate higher levels.

For now I only do higher levels on certain explicit cases, like when expanding `#[]` to `#at` and `#fetch`.

For now I think we should keep the explicitness in higher level, and think about global higher levels at the emitter root level.